src/styles/blocks/header.scss

```diff
@@ -0,0 +1,88 @@
+.header {
+  display: flex;
+  padding: 0 50px;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: white;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.logo {
+  display: block;
+  padding: 10px 0;
```

Comment:
Висоту header не можна встановити (it's not a good practice to hardcode height for elements with content inside. Just use padding)
. . . і паддинги зайві . 
а як тоді встановити висоту header ??

Reply:
а як тоді встановити точне позиціонування logo, якщо висоту жорстко не можна встановити, а падинги зайві?
inline  елемент я не встановлю так точно в 10 px від верхівки, та й напишіте, що дробові числа не можна використовувати (((

Reply:
use centering and flex or grid instead of hardcode position with `px`.  It' good for different screen sizes when your code centering elements dynamic according screen